src/CoreMedia/CMTime.cs

```diff
@@ -375,12 +376,14 @@ public static CMTime FromDictionary (NSDictionary dict)
 		}
 
 #if !XAMCORE_2_0
+#if !WATCH
 		[Obsolete ("Use 'FromDictionary (NSDictionary)' instead.")]
 		public static CMTime FromDictionary (IntPtr dict)
 		{
 			return CMTimeMakeFromDictionary (dict);
 		}
-#endif		
+#endif // !WATCH
+#endif	// !XAMCORE_@_)	
```

Comment:
`XAMCORE_@_` ?

Reply:
@dalexsoto brain doing funny things with the keyboard..